webapp/packages/plugin-connections/src/ConnectionAuthService.ts

```diff
@@ -56,12 +56,10 @@ export class ConnectionAuthService extends Dependency {
       return null;
     }
 
-    let connection = this.connectionInfoResource.get(key);
+    const connection = await this.connectionInfoResource.load(key, ['includeAuthNeeded', 'includeNetworkHandlersConfig', 'includeCredentialsSaved']);
```

Comment:
We probably want to load the ['includeAuthNeeded', 'includeNetworkHandlersConfig', 'includeCredentialsSaved'] includes only if the function is near the network handlers initialization. This would avoid loading useless data, for example, if we are already connected and the function merely returns the connection.